### PR TITLE
Fix/qa

### PR DESCRIPTION
**Pull request by paul-pelith**

### CODE REVIEW

**Comment by ytwang00** [comment]
Why there is no commit message, maybe u could squash all this commits with a commit message.

File: src/components/RedeemModal/index.tsx [hunk 1]
```diff
@@ -63,15 +63,19 @@ const RedeemModal = ({
...
+    : tryParseAmount('0');
 
   useEffect(() => {
-    if(unstakeState === TransactionState.SUCCESS && redeemModalOpen) {
-      toggleRedeemModal()
+    if (unstakeState === TransactionState.SUCCESS && redeemModalOpen) {
+      toggleRedeemModal();
     }
-  }, [unstakeState, redeemModalOpen]);
+    // redeemModalOpen should not be used here, because it will be reset to false after toggleRedeemModal
```
Comment (ytwang00) [inline src/components/RedeemModal/index.tsx:74]: Did this comment fixed?

Comment (ytwang00) [reply src/components/RedeemModal/index.tsx:74]: Or it is TODO:
?

File: src/components/RedeemModal/index.tsx [hunk 2]
```diff
@@ -83,7 +87,7 @@ const RedeemModal = ({
         <div sx={styles.hakkaBalanceContainer}>
           <span>Burn</span>
           <span>
-            sHAKKA Balance: {parseFloat(sHakkaBalance).toFixed(2) || '-'}
+            sHAKKA Balance: {(vault && parseFloat(formatUnits(vault.wAmount)).toFixed(4)) || '-'}
```
Comment (ytwang00) [inline src/components/RedeemModal/index.tsx:90]: formatUnits(vault.wAmount) used more than once, make it a variable

File: src/components/StakingPage/index.tsx [hunk 3]
```diff
@@ -200,6 +202,7 @@ const Staking = () => {
         <StakePositionTable
           data={vault}
           onRedeem={(index) => {
+            console.log('redeem', index)
```
Comment (ytwang00) [inline src/components/StakingPage/index.tsx]: should this be removed?

File: src/components/RedeemModal/index.tsx (hunk 1 above)
Comment (paul-pelith) [reply src/components/RedeemModal/index.tsx:74]: already fix

**Comment by paul-pelith** [comment]
reword commits

**Pull request merged**